recipe/meta.yaml

```diff
@@ -57,13 +57,12 @@ requirements:
     # Unix
     - {{ compiler('c') }}  # [unix]
     - {{ stdlib('c') }}  # [unix]
-    - {{ compiler('fortran') }}  # [unix]
+    - {{ compiler("fortran") }}
     - m4  # [unix]
     - make  # [unix]
-    # Win
-    - {{ compiler('m2w64_c') }}  # [win]
-    - {{ stdlib('m2w64_c') }}  # [win]
-    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ stdlib("c") }}
+    - {{ compiler("c") }}
+    - {{ compiler("fortran") }}
```

Comment:
This whole section is now duplicated with the unix compiler setup

Reply:
Yeah, I'll clean that up, too.